tests/unit/lms/views/admin/application_instance/update_test.py

```diff
@@ -105,11 +119,10 @@ def test_update_instance_saves_ai_settings(
     ):
         pyramid_request.params[f"{setting}.{sub_setting}"] = value
 
-        views.update_instance()
+        views.update_instance_settings()
```

Comment:
Tests were almost already separated between settings and the rest of the fields.